lib/algo/SnapshotDMD.h

```diff
@@ -0,0 +1,131 @@
+#ifndef included_SnapshotDMD_h
+#define included_SnapshotDMD_h
+
+#include "DMD.h"
+#include <vector>
+
+namespace CAROM {
+
+class Vector;
+class SnapshotDMD : public DMD
+{
+public:
+
+    /**
+     * @brief Constructor. Basic DMD with uniform time step size.
```

Comment:
Should the description include something specific to `SnapshotDMD`? It looks like this comment is copied from another class.

Reply:
This is a description of what the method is.  It does in fact assume uniform time steps between snapshots.  It is the same as the base DMD class, but so is its behavior.  The only difference between base DMD and snapshot DMD is in "train" and its ability to construct a new set of snapshots via "interpolateToNSnapshots".